app/assets/stylesheets/components/common/_news_signup_inpage.scss

```diff
@@ -0,0 +1,531 @@
+      margin-left: 33.333333333%;
+    }
+  }
+
+  &.news-signup-inpage--in-article .news-signup-inpage__container {
+    padding: 20px;
+  }
+
+  .news-signup-inpage__container__intro h3 {
+    @include body(22, 26);
+    color: $color-green-dark;
+    margin-top: 0;
+  }
+
+  .news-signup-inpage__container__intro p {
+    @include body(14, 14);
+  }
+
+  .news-signup-inpage__container__form input[type="email"] {
+    @include body(16, 16);
+    border: 1px solid $color-newsletter-input-border;
+    height: auto;
+    padding: $baseline-unit*2;
+    margin-right: $baseline-unit;
+    margin-bottom: $baseline-unit*2;
+
+    @include respond-to($mq-news-signup-inpage-article) {
+      width: 55%;
+      display: inline;
+      margin-bottom: $baseline-unit;
+    }
+  }
+
+  .news-signup-inpage__container__form .button {
+    width: 100%;
+    height: auto;
+    padding: 10px 12px;
+
+    @include respond-to($mq-news-signup-inpage-article) {
+      width: 40%;
+      float: right;
+    }
+  }
+
+  .news-signup-inpage__container__form label {
+    @include body(16, 24);
+    display: block;
+    font-weight: 500;
+    margin-bottom: $baseline-unit;
+  }
+
+  .news-signup-inpage__container__form p {
+    @include body(12, 12);
+    font-weight: 500;
+    clear: both;
+    margin-top: $baseline-unit*2;
+    margin-bottom: 0;
+  }
+
+  form.is-errored {
+    .news-signup-inpage__header {
+      .news-signup-inpage__text-container {
+
+        position: relative;
+        @include respond-to($mq-xs) {
+          min-height: 180px;
+        }
+        @include respond-to($mq-s) {
+          min-height: 170px;
+        }
+        @include respond-to($mq-m) {
+          min-height: 130px;
+        }
+
+        h3 {
+          @include respond-to($mq-m) {
+            display: none;
+          }
+        }
+
+        p {
+          display: none;
+        }
+
+        .news-signup-inpage__email-label {
+          @extend .is-errored;
+          @include body(22, 26);
+
+          border: none;
+          clip: auto;
+          width: auto;
+          height: auto;
+          margin: auto;
+          padding: initial;
+          overflow: auto;
+
+          position: absolute;
+          top: 50%;
+          left: 50%;
+          transform: translate(-50%, -50%);
+
+          @include respond-to($mq-xs) {
+            @include body(16, 22);
+            margin-top: 80px;
+            min-width: 215px;
+          }
+
+          @include respond-to($mq-s) {
+            @include body(16, 22);
+            margin-top: 70px;
+            min-width: 290px;
+          }
+
+          @include respond-to($mq-m) {
+            @include body(18, 22);
+            margin-top: 0;
+          }
+
+          @include respond-to($mq-3col-layout) {
+            min-width: 220px;
+          }
+
+          @include respond-to($mq-l) {
+            min-width: 280px;
+          }
+
+          @include respond-to($mq-xl) {
```

Comment:
These seem very specific mediaqueries, is there any way this can be simplified? No worries if not just curious.